File: 37-Python/Squash.py
from Campo import Campo
class Squash(Campo):
    def __init__(self, codice, sport, nomeCampo, larghezza, lunghezza, costoOrario, altezza, piano) -> None:
        super().__init__(codice, sport, nomeCampo, larghezza, lunghezza, costoOrario)
        self._altezza = altezza
        self._piano = piano
    
    def __str__(self) -> str:
        return super().__str__() +"\t-\t-\t"+ str(self._altezza) +"\t"+ str(self._piano) +"\t"+ str(self._costoOrario)

File: 37-Java/src/Squash.java
public class Squash extends Campo{
    private float altezza;
    private int piano;
    
    public Squash(int codice, String sport, String nomeCampo, float larghezza, float lunghezza, float costoOrario,
            float altezza, int piano) {
        super(codice, sport, nomeCampo, larghezza, lunghezza, costoOrario);
        this.altezza = altezza;
        this.piano = piano;
    }

    public String toString(){
        return super.toString() +"\t-\t-\t"+ altezza +"\t"+ piano +"\t"+ super.costoOrario;
    }
}
